assets/js/base/components/payment-methods/express-checkout.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useExpressPaymentMethods } from '@woocommerce/base-hooks';
+
+/**
+ * Internal dependencies
+ */
+import ExpressPaymentMethods from './express-payment-methods';
+
+import './style.scss';
+
+const ExpressCheckoutContainer = ( { children } ) => {
+	return (
+		<div className="wc-component__checkout-container wc-component__container-with-border">
+			<div className="wc-component__text-overlay-on-border">
+				<strong>
+					{ __( 'Express checkout', 'woo-gutenberg-products-block' ) }
+				</strong>
+			</div>
+			<div className="wc-component__container-content">{ children }</div>
+		</div>
+	);
+};
+
+const ExpressCheckoutFormControl = () => {
+	const { paymentMethods, isInitialized } = useExpressPaymentMethods();
+
+	// determine whether we even show this
+	// @todo if in the editor we probably would want to show a placeholder maybe?
+	if (
+		! isInitialized ||
+		( isInitialized && Object.keys( paymentMethods ).length === 0 )
```

Comment:
Hmm, not exactly.  `isInitialized` is true when every registered payment method has indicated it is initialized (might need to check for browser compatibility in the case of Apple Pay or https etc). So if this is false, there's still payment method's initializing.  It is _possible_ for there to be payment methods that have not initialized yet so I think the `isInitialized` check should remain.

The second condition here accounts for the scenario where all payment methods might have finished their initialization checks but are not a part of the final `paymentMethods` array because for one reason or another payments can not be made with them (pre-requisites aren't met). So we need to account for that scenario as well.

In a future pull (I think I already have the issue for it), I'll be implementing the placeholder here but what is used for the placeholder might vary.